.nvmrc

```diff
@@ -1 +1 @@
-12
+20.12.2
```

Comment:
I've gone back and forth on this, but I think as long as we are supporting Node 16 we should be developing on it to ensure we are not using new features or dependencies that are not supported on Node 16.

Reply:
This is only for default setting. It is not really used exact that version. I changed default to 16.20.2. But it is good that it is in github repo. All tests are passing on all 3 OSes and for nodejs 16, 18, 20, and also 22.